addons/point_of_sale/views/pos_config_view.xml

```diff
@@ -129,6 +117,21 @@
                                 </div>
                             </div>
                         </div>
+                        <div class="col-12 col-lg-6 o_setting_box">
+                            <field name="iface_start_categ_domain_ids" invisible="1"/>
```

Comment:
Sorry,I looked over this one. You don't need to put this field here any more since you did the calculation in the model.

Reply:
done !! :)